package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "start-rpc": "run-with-testrpc -m \"candy maple cake sugar pudding cream honey rich smooth crumble sweet treat\" --port 8544 --chainId 1337 --accounts 20 --gasLimit=10000000",
     "ganache": "ganache-cli -m \"candy maple cake sugar pudding cream honey rich smooth crumble sweet treat\" --port 8544 --accounts 20 --networkId=9 --gasLimit=10000000",
-    "prepare": "npm run build",
+    "prepare": "npm run build && husky install",
```

Comment:
Why `husky install` on `prepare`? It seems `prepare` is run before `npm install` and `npm publish`?

Reply:
I saw in [documentation](https://typicode.github.io/husky/#/) and on the [github page](https://github.com/typicode/husky#usage) that setting it on the `prepare script` enables git hooks directly after the install and long before `npm publish`

Reply:
The consequence of that is that at each commit, the eslint & prettier checks will be fired, and the commit wont be possible if there is any error.

Reply:
It will create husky.sh in .husky/_/

Reply:
Thanks!